modules/testsuite/cxf-tests/pom.xml

```diff
@@ -15,10 +15,10 @@
   
   <properties>
     <!-- This is used for test archives embedding Spring -->
-    <test.spring.version>3.0.3.RELEASE</test.spring.version>
+    <test.spring.version>5.1.5.RELEASE</test.spring.version>
     <!-- This is used for test archives embedding CXF and is intentionally different from the
          version specified in the jbossws-cxf main pom.xml to test there're no classloading clashes -->
-    <test.cxf.version>2.6.6</test.cxf.version>
+    <test.cxf.version>${cxf.version}</test.cxf.version>
```

Comment:
@jimma , have you read the comment here above? ("This ... is intentionally different from the version specified in the jbossws-cxf main pom.xml to test there're no classloading clashes")

Reply:
@asoldano  What is the classloading clashes issue ?  There is jboss-deployment-structure.xml to exclude webservice and weld subystems, then all cxf classes from jboss modules should not be loaded.

Reply:
@jimma right, if everything works as it should, it doesn't really matter. Actually I think when I did this I wanted to test a scenario closer to the real world (it would be strange that the cxf version embedded in the app is exactly the one that's already in the server) and which would definitely lead to something blowing up in case something of the webservices susbstytem is not properly excluded and leaks in the user deployment. Anyway, if there's valid reason for applying the changes in this PR, no problem with me :-)